config.php

```diff
@@ -37,15 +37,17 @@
 			'action' => 'comment',
 			'label' => 'question',
 			'comment' => <<<COMMENT
-_This is an automated comment, triggered by adding the label `question`._
-			
-Please note, that the GitHub Issue Tracker is for bug reports and feature requests only.
+Thank you for your question in our issue tracker. In order for this issue tracker to be effective,
+we ask our community to use our online resources specifically designed for support:
+* [our support forum](http://www.yiiframework.com/forum/)
+* [our IRC chat](http://www.yiiframework.com/chat/)
+* [our Gitter](https://gitter.im/yiisoft/yii2)
+* [our guide](http://www.yiiframework.com/doc-2.0/guide-index.html)
```

Comment:
"our" sounds weird here. I think the following sounds better:

- The support forum
- The IRC chat room
- The Gitter chat room
- The Definitive Guide to Yii